lib/tasks/events.rake

```diff
@@ -61,7 +63,17 @@ namespace :events do
       end
 
       if supplier_different
-        event.update(supplier_id: initial_version&.supplier_id) unless dry_run
+        ActiveRecord::Base.record_timestamps = false
+        ActiveRecord::Base.no_touching do
+          event_updated_at = event.updated_at
+          eventable_updated_at = event.eventable.updated_at
+
+          event.update(supplier_id: initial_version&.supplier_id) unless dry_run
```

Comment:
if we move to use `update_all` on a list of the events that match, we could avoid disabling things as that method  does not trigger Active Record callbacks or validations?

Reply:
I guess we could. Not sure what the advantage of skipping the validations would be?

Reply:
It is mainly to avoid the callbacks of the touch model and updated at without having to disable them explicitly, not the validations.

Reply:
However is validations a concern here though?

Reply:
spoke in person, won't use `update_all`